src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/relationships/relationships.js

```diff
@@ -113,195 +111,212 @@ class RelationshipsUI extends Component {
-                defaultMessage="Saved Search"
-              />);
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.search.calloutText"
-                defaultMessage="Here is the index pattern tied to this saved search."
-              />);
-            }
-            break;
-          case 'visualization':
-            if (type === 'index-pattern') {
-              calloutColor = 'success';
-              calloutTitle = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.visualization.indexPattern.calloutTitle"
-                defaultMessage="Index Pattern"
-              />);
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.visualization.indexPattern.calloutText"
-                defaultMessage="Here is the index pattern tied to this visualization."
-              />);
-            } else if (type === 'search') {
-              calloutColor = 'success';
-              calloutTitle = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.visualization.search.calloutTitle"
-                defaultMessage="Saved Search"
-              />);
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.visualization.search.calloutText"
-                defaultMessage="Here is the saved search tied to this visualization."
-              />);
-            } else {
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.visualization.calloutText"
-                defaultMessage="Here are some dashboards which contain this visualization. If
-                you delete this visualization, these dashboards will no longer
-                show them."
-              />);
-            }
-            break;
-          case 'index-pattern':
-            if (type === 'visualization') {
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.indexPattern.visualizations.calloutText"
-                defaultMessage="Here are some visualizations that use this index pattern. If
-                you delete this index pattern, these visualizations will not
-                longer work properly."
-              />);
-            } else if (type === 'search') {
-              calloutText = (<FormattedMessage
-                id="kbn.management.objects.objectsTable.relationships.indexPattern.searches.calloutText"
-                defaultMessage="Here are some saved searches that use this index pattern. If
-                you delete this index pattern, these saved searches will not
-                longer work properly."
-              />);
-            }
-            break;
-        }
+    const columns = [
```

Comment:
This is so much more readable than that for loop + switch statement from hell. Thanks for doing this!